website/src/pages/evaluate/rank_initial_prompts.tsx

```diff
@@ -0,0 +1,144 @@
+import { ArrowUpIcon, ArrowDownIcon } from "@heroicons/react/20/solid";
+import Head from "next/head";
+import { useState } from "react";
+import useSWRImmutable from "swr/immutable";
+import useSWRMutation from "swr/mutation";
+import clsx from "clsx";
+
+import fetcher from "src/lib/fetcher";
+import poster from "src/lib/poster";
+
+import { Button } from "src/components/Button";
+
+const RankInitialPrompts = () => {
+  const [tasks, setTasks] = useState([]);
+  const [ranking, setRanking] = useState<number[]>([]);
```

Comment:
Can you add the structure of this as a comment (basically what you wrote in the PR description)?  I think that'll help others that wander into looking at this.